src/angular/styles/_typography.scss

```diff
@@ -1888,177 +1689,121 @@ $tableStickyShadowWidth: 8;
 
-          @include mq($from: tabletPortrait) {
-            padding: $tableTbodyCellPadding;
+        @include mq($from: tabletPortrait) {
+          padding: pxToEm(18) pxToEm(16) pxToEm(19) pxToEm(16);
 
-            &:first-child {
-              border-left: $tableTheadPadding solid $tableBg;
-            }
+          &:first-child {
+            border-left: var(--sbb-table-thead-padding) solid var(--sbb-table-background-color);
+          }
 
-            &:last-child {
-              border-right: $tableTheadPadding solid $tableBg;
-            }
+          &:last-child {
+            border-right: var(--sbb-table-thead-padding) solid var(--sbb-table-background-color);
           }
         }
       }
     }
-
-    @include mq($from: desktop4k) {
-      font-size: pxToRem($sizeFontDefault4k);
-      line-height: pxToRem($sizeLineHeightDefault4k);
-    }
-    @include mq($from: desktop5k) {
-      font-size: pxToRem($sizeFontDefault5k);
-      line-height: pxToRem($sizeLineHeightDefault5k);
-    }
   }
 
-  html.sbb-lean & {
-    line-height: pxToRem($sizeLineHeightDefaultLean);
-
+  :where(html.sbb-lean) & {
     th {
-      color: $sbbColorGranite;
+      color: var(--sbb-color-granite);
     }
 
-    tbody,
-    tfoot {
-      > tr {
-        th,
-        td {
-          border-right: pxToRem(1) solid $sbbColorAluminum;
-        }
-      }
+    :is(tbody, tfoot) > tr :is(th, td) {
+      border-right: var(--sbb-border-width-thin) solid var(--sbb-color-aluminum);
     }
 
-    thead,
-    tbody,
-    tfoot {
-      > tr {
-        th,
-        td {
-          padding: pxToRem(6) $tableTbodyPadding pxToRem(6);
+    :is(thead, tbody, tfoot) > tr :is(th, td) {
+      padding: pxToRem(6) var(--sbb-table-tbody-padding) pxToRem(6);
 
-          &.sbb-table-group-with-next {
-            border-right: none;
-          }
+      &.sbb-table-group-with-next {
+        border-right: none;
+      }
 
-          &:first-child {
-            padding-left: $tableTbodyPadding;
-            border-left: none;
-          }
+      &:first-child {
+        padding-left: var(--sbb-table-tbody-padding);
+        border-left: none;
+      }
 
-          &:last-child {
-            padding-right: $tableTbodyPadding;
-            border-right: none;
-          }
-        }
+      &:last-child {
+        padding-right: var(--sbb-table-tbody-padding);
+        border-right: none;
       }
     }
 
     thead {
-      > tr:last-child {
-        th,
-        td {
-          border-bottom: pxToRem(1) solid $sbbColorGraphite;
-        }
+      > tr:last-child :is(th, td) {
+        border-bottom: var(--sbb-border-width-thin) solid var(--sbb-color-graphite);
       }
 
-      > tr {
-        th,
-        td {
-          border-right: pxToRem(1) solid $sbbColorGraphite;
+      > tr :is(th, td) {
+        border-right: var(--sbb-border-width-thin) solid var(--sbb-color-graphite);
 
+        :is(.sbb-table-wrapper-offset-right, .sbb-table-wrapper-offset-both)
```

Comment:
I think this term is not correct migrated: .sbb-table-wrapper-offset-right and .sbb-table-wrapper-offset-both is outside .sbb-table